test/test_torch.py

```diff
@@ -7761,19 +7761,9 @@ def cholesky_test_helper(n, batch_dims, upper):
     @dtypes(torch.float32, torch.float64, torch.complex64, torch.complex128)
     @tf32_on_and_off(0.01)
     def test_cholesky(self, device, dtype):
-        from torch.testing._internal.common_utils import \
-            (random_symmetric_pd_matrix,
-             random_fullrank_matrix_distinct_singular_value)
+        from torch.testing._internal.common_utils import random_hermitian_pd_matrix
 
-        # This is a workaround while there is no support for complex random_symmetric_pd_matrix
-        if dtype.is_complex:
```

Comment:
Is this being removed because you added support for complex random_symmetric_pd_matrix (I'm not sure what that is)? Is this because of your `lda` fix?

Reply:
Cholesky requires the input to be Hermitian positive definite, which is symmetric positive definite for the real-valued case. This is removed because I added `random_hermitian_pd_matrix`.
For real dtypes `random_hermitian_pd_matrix` is equivalent to `random_symmetric_pd_matrix` since `.conj()` does nothing.